src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -281,15 +291,17 @@ export function updateLayeredMaterialNodeElevation(context, layer, node, force)
             }
 
             node.layerUpdateState[layer.id].success();
-
             if (terrain.texture && terrain.texture.flipY) {
                 // DataTexture default to false, so make sure other Texture types
                 // do the same (eg image texture)
                 // See UV construction for more details
                 terrain.texture.flipY = false;
                 terrain.texture.needsUpdate = true;
             }
-            node.setTextureElevation(terrain);
+            if (!terrain.texture.image.data || checkNodeElevationTextureValidity(terrain.texture, layer.noDataValue))
```

Comment:
Maybe `checkXbilTextureValidity` would be more appropriate?

And what about calling: 
```js
node.layerUpdateState[layer.id].failure(Date.now(), true);
```
to indicate that this `layer` will never supply a valid texture for this node?